core/rawdb/database.go

```diff
@@ -350,37 +393,37 @@ func InspectDatabase(db ethdb.Database) error {
 	}
 	// Display the database statistic.
 	stats := [][]string{
-		{"Key-Value store", "Headers", headerSize.String()},
-		{"Key-Value store", "Bodies", bodySize.String()},
-		{"Key-Value store", "Receipts", receiptSize.String()},
-		{"Key-Value store", "Difficulties", tdSize.String()},
-		{"Key-Value store", "Block number->hash", numHashPairing.String()},
-		{"Key-Value store", "Block hash->number", hashNumPairing.String()},
-		{"Key-Value store", "Transaction index", txlookupSize.String()},
-		{"Key-Value store", "Bloombit index", bloomBitsSize.String()},
-		{"Key-Value store", "Contract codes", codeSize.String()},
-		{"Key-Value store", "Trie nodes", trieSize.String()},
-		{"Key-Value store", "Trie preimages", preimageSize.String()},
-		{"Key-Value store", "Account snapshot", accountSnapSize.String()},
-		{"Key-Value store", "Storage snapshot", storageSnapSize.String()},
-		{"Key-Value store", "Clique snapshots", cliqueSnapsSize.String()},
-		{"Key-Value store", "Singleton metadata", metadata.String()},
-		{"Ancient store", "Headers", ancientHeaders.String()},
-		{"Ancient store", "Bodies", ancientBodies.String()},
-		{"Ancient store", "Receipts", ancientReceipts.String()},
-		{"Ancient store", "Difficulties", ancientTds.String()},
-		{"Ancient store", "Block number->hash", ancientHashes.String()},
-		{"Light client", "CHT trie nodes", chtTrieNodes.String()},
-		{"Light client", "Bloom trie nodes", bloomTrieNodes.String()},
+		{"Key-Value store", "Headers", headerSize.String(), formatCounter(headerSizeCount)},
+		{"Key-Value store", "Bodies", bodySize.String(), formatCounter(bodySizeCount)},
+		{"Key-Value store", "Receipts", receiptSize.String(), formatCounter(receiptSizeCount)},
+		{"Key-Value store", "Difficulties", tdSize.String(), formatCounter(tdSizeCount)},
+		{"Key-Value store", "Block number->hash", numHashPairing.String(), formatCounter(numHashPairingCount)},
+		{"Key-Value store", "Block hash->number", hashNumPairing.String(), formatCounter(hashNumPairingCount)},
+		{"Key-Value store", "Transaction index", txlookupSize.String(), formatCounter(txlookupSizeCount)},
+		{"Key-Value store", "Bloombit index", bloomBitsSize.String(), formatCounter(bloomBitsSizeCount)},
+		{"Key-Value store", "Contract codes", codeSize.String(), formatCounter(codeSizeCount)},
+		{"Key-Value store", "Trie nodes", trieSize.String(), formatCounter(trieSizeCount)},
+		{"Key-Value store", "Trie preimages", preimageSize.String(), formatCounter(preimageSizeCount)},
+		{"Key-Value store", "Account snapshot", accountSnapSize.String(), formatCounter(accountSnapSizeCount)},
+		{"Key-Value store", "Storage snapshot", storageSnapSize.String(), formatCounter(storageSnapSizeCount)},
+		{"Key-Value store", "Clique snapshots", cliqueSnapsSize.String(), formatCounter(cliqueSnapsSizeCount)},
+		{"Key-Value store", "Singleton metadata", metadata.String(), formatCounter(metadataCount)},
+		{"Ancient store", "Headers", ancientHeaders.String(), "NA"},
```

Comment:
These should be fairly easy to obtain too, by calling `Ancients()` above where it currently only calls `AncientSize()`. Needs only be done once for the whole group

Reply:
I see, nice! I'll work on it.